Only in /Users/Dustin/www/medicalidfashions.boi/www/: .git
diff -uprb a/.gitignore b/.gitignore
--- .gitignore	2015-08-19 17:20:31.000000000 -0400
+++ .gitignore	2015-09-14 10:30:12.000000000 -0400
@@ -1,6 +1,12 @@
 # Ignore configuration files that may contain sensitive information.
 sites/*/settings*.php

+# Ignore .DS_Store
+*.DS_Store
+
+# Ignore DB dumps
+*.sql
+
 # Ignore paths that contain user-generated content.
 sites/*/files
 sites/*/private
Only in /Users/Dustin/www/medicalidfashions.boi/www/: drushrc.php
Only in /Users/Dustin/www/medicalidfashions.boi/www/: error_log
Only in /Users/Dustin/www/medicalidfashions.boi/www/: live.sql
Only in /Users/Dustin/www/medicalidfashions.boi/www/modules: .DS_Store
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites: .DS_Store
Only in /tmp/hacked-cache-Dustin/hackedProjectWebFilesDownloader/core/drupal/7.39/drupal-7.39/sites: README.txt
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites/all: .DS_Store
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites/all: README.txt
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites/all/libraries: .DS_Store
Only in /tmp/hacked-cache-Dustin/hackedProjectWebFilesDownloader/core/drupal/7.39/drupal-7.39/sites/all/libraries: README.txt
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites/all/libraries: braintree_php
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites/all/libraries: ckeditor
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites/all/libraries: colorbox
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites/all/libraries: elevatezoom
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites/all/libraries: flexslider
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites/all/libraries: fpdi
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites/all/libraries: jquery.nivo.slider.js
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites/all/libraries: pap
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites/all/modules: ._.DS_Store
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites/all/modules: ._medidf_structure_feature
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites/all/modules: .gitignore
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites/all/modules: contrib
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites/all/modules: custom
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites/all: patches
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites/all/themes: .DS_Store
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites/all/themes: .idea
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites/all/themes: adminimal_theme
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites/all/themes: medicalidfashions_theme
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites/all/themes: sankofa
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites/all/themes: zen
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites/all/themes: zurb_foundation
diff -uprb a/sites/default/default.settings.php b/sites/default/default.settings.php
--- sites/default/default.settings.php	2015-08-19 17:20:31.000000000 -0400
+++ sites/default/default.settings.php	2015-09-07 15:35:49.000000000 -0400
@@ -5,51 +5,41 @@
  * Drupal site-specific configuration file.
  *
  * IMPORTANT NOTE:
- * This file may have been set to read-only by the Drupal installation program.
- * If you make changes to this file, be sure to protect it again after making
- * your modifications. Failure to remove write permissions to this file is a
- * security risk.
- *
- * The configuration file to be loaded is based upon the rules below. However
- * if the multisite aliasing file named sites/sites.php is present, it will be
- * loaded, and the aliases in the array $sites will override the default
- * directory rules below. See sites/example.sites.php for more information about
- * aliases.
- *
- * The configuration directory will be discovered by stripping the website's
- * hostname from left to right and pathname from right to left. The first
- * configuration file found will be used and any others will be ignored. If no
- * other configuration file is found then the default configuration file at
- * 'sites/default' will be used.
+ * This file may have been set to read-only by the Drupal installation
+ * program. If you make changes to this file, be sure to protect it again
+ * after making your modifications. Failure to remove write permissions
+ * to this file is a security risk.
+ *
+ * The configuration file to be loaded is based upon the rules below.
+ *
+ * The configuration directory will be discovered by stripping the
+ * website's hostname from left to right and pathname from right to
+ * left. The first configuration file found will be used and any
+ * others will be ignored. If no other configuration file is found
+ * then the default configuration file at 'sites/default' will be used.
  *
  * For example, for a fictitious site installed at
- * http://www.drupal.org:8080/mysite/test/, the 'settings.php' file is searched
- * for in the following directories:
+ * http://www.drupal.org/mysite/test/, the 'settings.php'
+ * is searched in the following directories:
  *
- * - sites/8080.www.drupal.org.mysite.test
  * - sites/www.drupal.org.mysite.test
  * - sites/drupal.org.mysite.test
  * - sites/org.mysite.test
  *
- * - sites/8080.www.drupal.org.mysite
  * - sites/www.drupal.org.mysite
  * - sites/drupal.org.mysite
  * - sites/org.mysite
  *
- * - sites/8080.www.drupal.org
  * - sites/www.drupal.org
  * - sites/drupal.org
  * - sites/org
  *
  * - sites/default
  *
- * Note that if you are installing on a non-standard port number, prefix the
+ * If you are installing on a non-standard port number, prefix the
  * hostname with that number. For example,
  * http://www.drupal.org:8080/mysite/test/ could be loaded from
  * sites/8080.www.drupal.org.mysite.test/.
- *
- * @see example.sites.php
- * @see conf_path()
  */

 /**
@@ -83,13 +73,11 @@
  * webserver.  For most other drivers, you must specify a
  * username, password, host, and database name.
  *
- * Transaction support is enabled by default for all drivers that support it,
- * including MySQL. To explicitly disable it, set the 'transactions' key to
- * FALSE.
- * Note that some configurations of MySQL, such as the MyISAM engine, don't
- * support it and will proceed silently even if enabled. If you experience
- * transaction related crashes with such configuration, set the 'transactions'
- * key to FALSE.
+ * Some database engines support transactions.  In order to enable
+ * transaction support for a given database, set the 'transactions' key
+ * to TRUE.  To disable it, set it to FALSE.  Note that the default value
+ * varies by driver.  For MySQL, the default is FALSE since MyISAM tables
+ * do not support transactions.
  *
  * For each database, you may optionally specify multiple "target" databases.
  * A target database allows Drupal to try to send certain queries to a
@@ -149,7 +137,7 @@
  *     'authmap'   => 'shared_',
  *   ),
  * @endcode
- * You can also use a reference to a schema/database as a prefix. This may be
+ * You can also use a reference to a schema/database as a prefix. This maybe
  * useful if your Drupal installation exists in a schema that is not the default
  * or you want to access several databases from the same code base at the same
  * time.
@@ -273,8 +261,8 @@ $drupal_hash_salt = '';
  *
  * To see what PHP settings are possible, including whether they can be set at
  * runtime (by using ini_set()), read the PHP documentation:
- * http://www.php.net/manual/ini.list.php
- * See drupal_environment_initialize() in includes/bootstrap.inc for required
+ * http://www.php.net/manual/en/ini.list.php
+ * See drupal_initialize_variables() in includes/bootstrap.inc for required
  * runtime settings and the .htaccess file for non-runtime settings. Settings
  * defined there should not be duplicated here so as to avoid conflict issues.
  */
@@ -309,7 +297,7 @@ ini_set('session.cookie_lifetime', 20000
  * output filter may not have sufficient memory to process it.  If you
  * experience this issue, you may wish to uncomment the following two lines
  * and increase the limits of these variables.  For more information, see
- * http://php.net/manual/pcre.configuration.php.
+ * http://php.net/manual/en/pcre.configuration.php.
  */
 # ini_set('pcre.backtrack_limit', 200000);
 # ini_set('pcre.recursion_limit', 200000);
@@ -435,21 +423,9 @@ ini_set('session.cookie_lifetime', 20000
 # $conf['js_gzip_compression'] = FALSE;

 /**
- * Block caching:
- *
- * Block caching may not be compatible with node access modules depending on
- * how the original block cache policy is defined by the module that provides
- * the block. By default, Drupal therefore disables block caching when one or
- * more modules implement hook_node_grants(). If you consider block caching to
- * be safe on your site and want to bypass this restriction, uncomment the line
- * below.
- */
-# $conf['block_cache_bypass_node_grants'] = TRUE;
-
-/**
  * String overrides:
  *
- * To override specific strings on your site with or without enabling the Locale
+ * To override specific strings on your site with or without enabling locale
  * module, add an entry to this list. This functionality allows you to change
  * a small number of your site's default English language interface strings.
  *
@@ -504,64 +480,33 @@ ini_set('session.cookie_lifetime', 20000
  */
 $conf['404_fast_paths_exclude'] = '/\/(?:styles)\//';
 $conf['404_fast_paths'] = '/\.(?:txt|png|gif|jpe?g|css|js|ico|swf|flv|cgi|bat|pl|dll|exe|asp)$/i';
-$conf['404_fast_html'] = '<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML+RDFa 1.0//EN" "http://www.w3.org/MarkUp/DTD/xhtml-rdfa-1.dtd"><html xmlns="http://www.w3.org/1999/xhtml"><head><title>404 Not Found</title></head><body><h1>Not Found</h1><p>The requested URL "@path" was not found on this server.</p></body></html>';
+$conf['404_fast_html'] = '<html xmlns="http://www.w3.org/1999/xhtml"><head><title>404 Not Found</title></head><body><h1>Not Found</h1><p>The requested URL "@path" was not found on this server.</p></body></html>';

 /**
- * By default the page request process will return a fast 404 page for missing
- * files if they match the regular expression set in '404_fast_paths' and not
- * '404_fast_paths_exclude' above. 404 errors will simultaneously be logged in
- * the Drupal system log.
- *
- * You can choose to return a fast 404 page earlier for missing pages (as soon
- * as settings.php is loaded) by uncommenting the line below. This speeds up
- * server response time when loading 404 error pages and prevents the 404 error
- * from being logged in the Drupal system log. In order to prevent valid pages
- * such as image styles and other generated content that may match the
- * '404_fast_paths' regular expression from returning 404 errors, it is
- * necessary to add them to the '404_fast_paths_exclude' regular expression
- * above. Make sure that you understand the effects of this feature before
- * uncommenting the line below.
+ * By default, fast 404s are returned as part of the normal page request
+ * process, which will properly serve valid pages that happen to match and will
+ * also log actual 404s to the Drupal log. Alternatively you can choose to
+ * return a 404 now by uncommenting the following line. This will reduce server
+ * load, but will cause even valid pages that happen to match the pattern to
+ * return 404s, rather than the actual page. It will also prevent the Drupal
+ * system log entry. Ensure you understand the effects of this before enabling.
+ *
+ * To enable this functionality, remove the leading hash sign below.
  */
 # drupal_fast_404();

 /**
- * External access proxy settings:
- *
- * If your site must access the Internet via a web proxy then you can enter
- * the proxy settings here. Currently only basic authentication is supported
- * by using the username and password variables. The proxy_user_agent variable
- * can be set to NULL for proxies that require no User-Agent header or to a
- * non-empty string for proxies that limit requests to a specific agent. The
- * proxy_exceptions variable is an array of host names to be accessed directly,
- * not via proxy.
- */
-# $conf['proxy_server'] = '';
-# $conf['proxy_port'] = 8080;
-# $conf['proxy_username'] = '';
-# $conf['proxy_password'] = '';
-# $conf['proxy_user_agent'] = '';
-# $conf['proxy_exceptions'] = array('127.0.0.1', 'localhost');
-
-/**
  * Authorized file system operations:
  *
  * The Update manager module included with Drupal provides a mechanism for
  * site administrators to securely install missing updates for the site
- * directly through the web user interface. On securely-configured servers,
- * the Update manager will require the administrator to provide SSH or FTP
- * credentials before allowing the installation to proceed; this allows the
- * site to update the new files as the user who owns all the Drupal files,
- * instead of as the user the webserver is running as. On servers where the
- * webserver user is itself the owner of the Drupal files, the administrator
- * will not be prompted for SSH or FTP credentials (note that these server
- * setups are common on shared hosting, but are inherently insecure).
- *
- * Some sites might wish to disable the above functionality, and only update
- * the code directly via SSH or FTP themselves. This setting completely
+ * directly through the web user interface by providing either SSH or FTP
+ * credentials. This allows the site to update the new files as the user who
+ * owns all the Drupal files, instead of as the user the webserver is running
+ * as. However, some sites might wish to disable this functionality, and only
+ * update the code directly via SSH or FTP themselves. This setting completely
  * disables all functionality related to these authorized file operations.
  *
- * @see http://drupal.org/node/244924
- *
  * Remove the leading hash signs to disable.
  */
 # $conf['allow_authorize_operations'] = FALSE;
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites/default: files
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites/default: settings.php
diff -uprb a/sites/example.sites.php b/sites/example.sites.php
--- sites/example.sites.php	2015-08-19 17:20:31.000000000 -0400
+++ sites/example.sites.php	2015-09-07 15:35:49.000000000 -0400
@@ -4,52 +4,40 @@
  * @file
  * Configuration file for Drupal's multi-site directory aliasing feature.
  *
- * This file allows you to define a set of aliases that map hostnames, ports, and
- * pathnames to configuration directories in the sites directory. These aliases
- * are loaded prior to scanning for directories, and they are exempt from the
- * normal discovery rules. See default.settings.php to view how Drupal discovers
- * the configuration directory when no alias is found.
- *
- * Aliases are useful on development servers, where the domain name may not be
- * the same as the domain of the live server. Since Drupal stores file paths in
- * the database (files, system table, etc.) this will ensure the paths are
- * correct when the site is deployed to a live server.
+ * Drupal searches for an appropriate configuration directory based on the
+ * website's hostname and pathname. A detailed description of the rules for
+ * discovering the configuration directory can be found in the comment
+ * documentation in 'sites/default/default.settings.php'.
+ *
+ * This file allows you to define a set of aliases that map hostnames and
+ * pathnames to configuration directories. These aliases are loaded prior to
+ * scanning for directories, and they are exempt from the normal discovery
+ * rules. The aliases are defined in an associative array named $sites, which
+ * should look similar to the following:
+ *
+ * $sites = array(
+ *   'devexample.com' => 'example.com',
+ *   'localhost.example' => 'example.com',
+ * );
+ *
+ * The above array will cause Drupal to look for a directory named
+ * "example.com" in the sites directory whenever a request comes from
+ * "example.com", "devexample.com", or "localhost/example". That is useful
+ * on development servers, where the domain name may not be the same as the
+ * domain of the live server. Since Drupal stores file paths into the database
+ * (files, system table, etc.) this will ensure the paths are correct while
+ * accessed on development servers.
  *
  * To use this file, copy and rename it such that its path plus filename is
  * 'sites/sites.php'. If you don't need to use multi-site directory aliasing,
  * then you can safely ignore this file, and Drupal will ignore it too.
+ */
+
+/**
+ * Multi-site directory aliasing:
  *
- * Aliases are defined in an associative array named $sites. The array is
- * written in the format: '<port>.<domain>.<path>' => 'directory'. As an
- * example, to map http://www.drupal.org:8080/mysite/test to the configuration
- * directory sites/example.com, the array should be defined as:
- * @code
- * $sites = array(
- *   '8080.www.drupal.org.mysite.test' => 'example.com',
- * );
- * @endcode
- * The URL, http://www.drupal.org:8080/mysite/test/, could be a symbolic link or
- * an Apache Alias directive that points to the Drupal root containing
- * index.php. An alias could also be created for a subdomain. See the
- * @link http://drupal.org/documentation/install online Drupal installation guide @endlink
- * for more information on setting up domains, subdomains, and subdirectories.
- *
- * The following examples look for a site configuration in sites/example.com:
- * @code
- * URL: http://dev.drupal.org
- * $sites['dev.drupal.org'] = 'example.com';
- *
- * URL: http://localhost/example
- * $sites['localhost.example'] = 'example.com';
- *
- * URL: http://localhost:8080/example
- * $sites['8080.localhost.example'] = 'example.com';
- *
- * URL: http://www.drupal.org:8080/mysite/test/
- * $sites['8080.www.drupal.org.mysite.test'] = 'example.com';
- * @endcode
- *
- * @see default.settings.php
- * @see conf_path()
- * @see http://drupal.org/documentation/install/multi-site
+ * Edit the lines below to define directory aliases. Remove the leading hash
+ * signs to enable.
  */
+# $sites['devexample.com'] = 'example.com';
+# $sites['localhost.example'] = 'example.com';
Only in /Users/Dustin/www/medicalidfashions.boi/www/sites: sites.php
