diff -uprb a/imagezoom.module b/imagezoom.module
--- imagezoom.module	2014-12-13 17:21:26.000000000 -0500
+++ imagezoom.module	2015-07-30 15:48:40.000000000 -0400
@@ -19,13 +19,13 @@ function imagezoom_libraries_info() {
     'vendor url' => 'http://www.elevateweb.co.uk/image-zoom',
     'download url' => 'http://www.elevateweb.co.uk/image-zoom/download',
     'version arguments' => array(
-      'file' => 'jquery.elevateZoom.js',
+      'file' => 'jquery.elevatezoom.js',
       'pattern' => '@elevateZoom\s+([0-9\.]+)@',
       'lines' => 2,
       'cols' => 50,
     ),
     'files' => array(
-      'js' => array('jquery.elevateZoom.js'),
+      'js' => array('jquery.elevatezoom.js'),
     ),
   );

